diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/comments/HeaderCommentRule.kt

```diff
@@ -190,6 +190,41 @@ class HeaderCommentRule(private val configRules: List<RulesConfig>) : Rule("head
         }
     }
 
+    /**
+     * Deletes all spaces and newlines
+     * Used to compare copyrights in yaml and file
+     */
+    private fun String.flatten(): String =
+        replace("\n","")
+        .replace(" ","")
+
+    /**
+     * If it is multiline copyright, this method makes needed indents.
+     * Otherwise, if it is one line copyright, it returns it trimmed.
+     */
+    private fun handleMultilineCopyright(copyrightText: String): String {
+        if (copyrightText.contains("\n")) {
+            return copyrightText
+                    .lines()
+                    .reduce { acc, nextLine ->
+                        when {
+                            !nextLine.any { it in 'a'..'z' || it in 'A'.. 'Z' } -> {
```

Comment:
- instead of `!any` you should use `none`
- what if there are digits or other symbols? I guess you could simply use `String.isBlank`?